skysat/ndvi/README.md

```diff
@@ -28,7 +28,7 @@ NDVI in certain intervals to RGBA (RGB+Alpha) values for visualization.
 
 Small example of NDVI visualized.
```

Comment:
I would change this to:

NDVI visualisation over Rome, Italy, acquired on xx.xx.2022.